src/modules/organizations/containers/details-form-container.jsx

```diff
@@ -93,14 +95,16 @@ class DetailsFormContainer extends React.Component {
           <fieldset className="form__fieldset">
             <label className="form__label" htmlFor="introduction">
               Introduction
-              <MarkdownEditor
-                id="introduction"
-                name="introduction"
-                onChange={this.handleTextAreaChange}
-                project={this.props.organization}
-                rows="10"
-                value={this.state.textarea}
-              />
+              <div className="form__input--top-margin">
+                <MarkdownEditor
+                  id="introduction"
```

Comment:
You can add `className="form__input--top-margin"` here, which avoids the awakward wrapping `div`.